multinet/db.py

```diff
@@ -118,7 +118,7 @@ def workspace_mapping_collection() -> StandardCollection:
 
 # Caches the document that maps an external workspace name to it's internal one
 @lru_cache()
-def workspace_mapping(name: str) -> Optional[WorkspaceDocument]:
+def workspace_mapping(name: str) -> Optional[Dict]:
```

Comment:
I tried to avoid dealing with `TypedDict`s in the first pass, since that was where much of the pain points came from originally. If we can deal with them in a sane way I'm willing to add them, but I think we need to come up with a methodology for using them.

Reply:
Let's worry about this in a followup. I can take it on myself: #460.